### ISSUE TITLE
Working with Symfony 5

### ISSUE THREAD

**Issue by aimproxy** [marcw/twig-heroicons#1]
How this exactly is supposed to work? Do I have to register something in Symfony Services to get it working?

```
An exception has been thrown during the rendering of a template ("Heroicon "academy-cap" in style "outline" cannot be found or is not readable.").
```

```twig
{% extends '@EasyAdmin/page/content.html.twig' %}
{% block page_content %}
    <div class="px-4 py-5 my-5 text-center">
        {{ heroicon('academy-cap', '', 'outline') }}
```

**Comment by aimproxy** [comment]
I would suggest that the https://github.com/tailwindlabs/heroicons repo was a sub project of this one. The benefits are clear, everyone gets the latest svg from the official heroicons repo! By the way thx for the library!

**Comment by marcw** [comment]
fixed by #2
